atest/run.py

```diff
@@ -53,9 +57,9 @@
 '''.strip()
 
 
-def atests(interpreter, *arguments):
+def atests(coverage, interpreter, *arguments):
```

Comment:
I'd prefer `def atest(interpreter, *arguments, coverage=False)` here. This is code is Python 3 only so keyword-only arguments are fine.